src/agents/models/openai_responses.py

```diff
@@ -208,8 +208,10 @@ async def _fetch_response(
         list_input = ItemHelpers.input_to_new_input_list(input)
 
         parallel_tool_calls = (
```

Comment:
this would be easier to read as an if/elif/else block rather than this expression